packages/web/src/components/admin_page/hackathon/hackathon_mentors/render_mentors.jsx

```diff
@@ -12,7 +12,7 @@ const RenderMentors = () => {
     const history = useNavigate();
     const [mentors, setMentors] = useState();
 
-    const getJobs = () => {
+    const getMentors = () => {
         axios({
             method: 'get',
             url: url + '/api/mentors'
```

Comment:
catch the exception here otherwise we get an additional error log which is not supposed to be in the console

Reply:
So to just move the .catch(err) -> { ... } code within axios?

Reply:
I don't quite catch what you mean, but I will handle it myself.